web/src/message_live_update.js

```diff
@@ -8,19 +8,6 @@ export function rerender_messages_view() {
     }
 }
 
-export function rerender_messages_view_by_message_ids(message_ids) {
-    const messages_to_render = [];
-    for (const id of message_ids) {
-        const message = message_store.get(id);
-        if (message !== undefined) {
-            messages_to_render.push(message);
-        }
-    }
-    for (const list of message_lists.all_rendered_message_lists()) {
-        list.view.rerender_messages(messages_to_render);
-    }
-}
```

Comment:
This may be a useful helper for other applications, even though it's not the right choice here, so I skipped deleting it.